### PR TITLE
Add Folks Finance yield

### PR DESCRIPTION
**Pull request by pf55351**

### CODE REVIEW

File: src/adaptors/package.json [hunk 1]
```diff
@@ -20,6 +20,9 @@
     "dotenv": "^10.0.0",
     "graphql": "^15.5.1",
     "graphql-request": "^3.5.0",
+    "hi-base32": "^0.5.1",
+    "js-sha512": "^0.8.0",
```
Comment (slasher125) [inline src/adaptors/package.json]: are `hi-base32` and `js-sha512` actually used anywhere in your adapter? if not, pls remove

File: package.json [hunk 2]
```diff
@@ -1,20 +1,14 @@
 {
-  "name": "defillama-apy-server",
```
Comment (slasher125) [inline package.json]: please revert all changes in here, this is not supposed to be updated

File: src/adaptors/package.json (hunk 1 above)
Comment (pf55351) [reply src/adaptors/package.json]: Removed

File: package.json (hunk 2 above)
Comment (pf55351) [reply package.json]: Reverted

**Comment by pf55351** [comment]
Package "limiter" is not found but it is necessary... how can i add it?
Thanks

**Comment by slasher125** [comment]
> Package "limiter" is not found but it is necessary... how can i add it? Thanks

- gm, sry, sure u can add that one back to the top lvl package.json without changing the other stuff.
- could u pls add `ltv` so we can display u on our borrow pages. ty

**Comment by llamatester** [comment]
The folks-finance-lending adapter exports pools: 
        
 
 Test Suites: 1 passed, 1 total
    Tests:       56 passed, 56 total
    Snapshots:   0 total
    Time:        0.465 s
    Ran all test suites.
    
    Nb of pools: 10
     
    
    Sample pools:
    ┌─────────┬───────────────────────┬────────────┬─────────────────────────┬─────────┬─────────────┬────────────────┬────────────────┬─────────┬───────────────┬───────────┬──────────────┐
    │ (index) │         pool          │   chain    │         project         │ symbol  │   tvlUsd    │ totalSupplyUsd │ totalBorrowUsd │ apyBase │ apyBaseBorrow │ apyReward │ rewardTokens │
    ├─────────┼───────────────────────┼────────────┼─────────────────────────┼─────────┼─────────────┼────────────────┼────────────────┼─────────┼───────────────┼───────────┼──────────────┤
    │    0    │ '971373611-algorand'  │ 'Algorand' │ 'folks-finance-lending' │ 'GOETH' │   93049.1   │   100729.84    │    7680.74     │  0.07   │     1.05      │           │              │
    │    1    │ '971373361-algorand'  │ 'Algorand' │ 'folks-finance-lending' │ 'GOBTC' │  202850.58  │   203782.64    │     932.05     │    0    │     0.06      │           │              │
    │    2    │ '1067289273-algorand' │ 'Algorand' │ 'folks-finance-lending' │ 'WBTC'  │    35.31    │     37.29      │      1.97      │  0.09   │     0.72      │           │              │
    │    3    │ '1060585819-algorand' │ 'Algorand' │ 'folks-finance-lending' │ 'GARD'  │  25028.65   │    26858.06    │    1829.41     │  0.14   │     2.23      │           │              │
    │    4    │ '1044267181-algorand' │ 'Algorand' │ 'folks-finance-lending' │ 'OPUL'  │  140975.94  │   180287.32    │    39311.38    │  0.66   │     3.43      │           │              │
    │    5    │ '971370097-algorand'  │ 'Algorand' │ 'folks-finance-lending' │ 'GALGO' │ 11195409.78 │  11195409.78   │       0        │    0    │       0       │           │              │
    │    6    │ '1067289481-algorand' │ 'Algorand' │ 'folks-finance-lending' │ 'WETH'  │    3.59     │      4.35      │      0.76      │  0.38   │     2.42      │           │              │
    │    7    │ '971368268-algorand'  │ 'Algorand' │ 'folks-finance-lending' │ 'ALGO'  │  2012763.5  │   9555787.67   │   7543024.18   │  8.14   │     11.49     │           │              │
    │    8    │ '971372700-algorand'  │ 'Algorand' │ 'folks-finance-lending' │ 'USDT'  │  437967.62  │   490472.84    │    52505.21    │  0.24   │     2.49      │   19.76   │   [ '0' ]    │
    │    9    │ '971372237-algorand'  │ 'Algorand' │ 'folks-finance-lending' │ 'USDC'  │ 1187824.37  │   1687818.45   │   499994.08    │  1.01   │      3.8      │   15.76   │   [ '0' ]    │
    └─────────┴───────────────────────┴────────────┴─────────────────────────┴─────────┴─────────────┴────────────────┴────────────────┴─────────┴───────────────┴───────────┴──────────────┘
    This adapter contains some pools with <10k TVL, these pools won't be shown in DefiLlama

**Comment by llamatester** [comment]
The folks-finance-lending adapter exports pools: 
        
 
 Test Suites: 1 passed, 1 total
    Tests:       56 passed, 56 total
    Snapshots:   0 total
    Time:        0.357 s
    Ran all test suites.
    
    Nb of pools: 10
     
    
    Sample pools:
    ┌─────────┬───────────────────────┬────────────┬─────────────────────────┬─────────┬─────────────┬────────────────┬────────────────┬─────────┬───────────────┬─────┬───────────┬──────────────┐
    │ (index) │         pool          │   chain    │         project         │ symbol  │   tvlUsd    │ totalSupplyUsd │ totalBorrowUsd │ apyBase │ apyBaseBorrow │ ltv │ apyReward │ rewardTokens │
    ├─────────┼───────────────────────┼────────────┼─────────────────────────┼─────────┼─────────────┼────────────────┼────────────────┼─────────┼───────────────┼─────┼───────────┼──────────────┤
    │    0    │ '971368268-algorand'  │ 'Algorand' │ 'folks-finance-lending' │ 'ALGO'  │ 2021434.96  │   9587282.7    │   7565847.74   │  8.13   │     11.49     │ 65  │           │              │
    │    1    │ '971370097-algorand'  │ 'Algorand' │ 'folks-finance-lending' │ 'GALGO' │ 11228030.56 │  11228030.56   │       0        │    0    │       0       │ 65  │           │              │
    │    2    │ '1044267181-algorand' │ 'Algorand' │ 'folks-finance-lending' │ 'OPUL'  │  141727.32  │   181248.22    │    39520.9     │  0.66   │     3.43      │  0  │           │              │
    │    3    │ '1060585819-algorand' │ 'Algorand' │ 'folks-finance-lending' │ 'GARD'  │  25028.23   │    26857.62    │    1829.38     │  0.14   │     2.23      │ 70  │           │              │
    │    4    │ '971373361-algorand'  │ 'Algorand' │ 'folks-finance-lending' │ 'GOBTC' │  203831.26  │   204766.83    │     935.57     │    0    │     0.06      │ 70  │           │              │
    │    5    │ '1067289273-algorand' │ 'Algorand' │ 'folks-finance-lending' │ 'WBTC'  │    35.44    │     37.43      │      1.98      │  0.09   │     0.72      │ 70  │           │              │
    │    6    │ '971373611-algorand'  │ 'Algorand' │ 'folks-finance-lending' │ 'GOETH' │  93391.11   │   101100.08    │    7708.97     │  0.07   │     1.05      │ 70  │           │              │
    │    7    │ '1067289481-algorand' │ 'Algorand' │ 'folks-finance-lending' │ 'WETH'  │     3.6     │      4.37      │      0.77      │  0.38   │     2.42      │ 70  │           │              │
    │    8    │ '971372700-algorand'  │ 'Algorand' │ 'folks-finance-lending' │ 'USDT'  │  438003.69  │   490513.23    │    52509.54    │  0.24   │     2.49      │  0  │   19.82   │   [ '0' ]    │
    │    9    │ '971372237-algorand'  │ 'Algorand' │ 'folks-finance-lending' │ 'USDC'  │ 1184698.45  │   1685022.29   │   500323.84    │  1.01   │      3.8      │ 80  │   15.84   │   [ '0' ]    │
    └─────────┴───────────────────────┴────────────┴─────────────────────────┴─────────┴─────────────┴────────────────┴────────────────┴─────────┴───────────────┴─────┴───────────┴──────────────┘
    This adapter contains some pools with <10k TVL, these pools won't be shown in DefiLlama

**Comment by llamatester** [comment]
The folks-finance-lending adapter exports pools: 
        
 
 Test Suites: 1 passed, 1 total
    Tests:       56 passed, 56 total
    Snapshots:   0 total
    Time:        0.475 s
    Ran all test suites.
    
    Nb of pools: 10
     
    
    Sample pools:
    ┌─────────┬───────────────────────┬────────────┬─────────────────────────┬─────────┬─────────────┬────────────────┬────────────────┬─────────┬───────────────┬──────┬───────────┬──────────────┐
    │ (index) │         pool          │   chain    │         project         │ symbol  │   tvlUsd    │ totalSupplyUsd │ totalBorrowUsd │ apyBase │ apyBaseBorrow │ ltv  │ apyReward │ rewardTokens │
    ├─────────┼───────────────────────┼────────────┼─────────────────────────┼─────────┼─────────────┼────────────────┼────────────────┼─────────┼───────────────┼──────┼───────────┼──────────────┤
    │    0    │ '971373611-algorand'  │ 'Algorand' │ 'folks-finance-lending' │ 'GOETH' │  93433.43   │   101145.89    │    7712.46     │  0.07   │     1.05      │ 0.7  │           │              │
    │    1    │ '1067289481-algorand' │ 'Algorand' │ 'folks-finance-lending' │ 'WETH'  │     3.6     │      4.37      │      0.77      │  0.38   │     2.42      │ 0.7  │           │              │
    │    2    │ '1067289273-algorand' │ 'Algorand' │ 'folks-finance-lending' │ 'WBTC'  │    35.48    │     37.46      │      1.98      │  0.09   │     0.72      │ 0.7  │           │              │
    │    3    │ '971368268-algorand'  │ 'Algorand' │ 'folks-finance-lending' │ 'ALGO'  │ 2026648.79  │   9609909.83   │   7583261.05   │  8.13   │     11.49     │ 0.65 │           │              │
    │    4    │ '1060585819-algorand' │ 'Algorand' │ 'folks-finance-lending' │ 'GARD'  │  25019.47   │    26821.4     │    1801.92     │  0.13   │     2.22      │ 0.7  │           │              │
    │    5    │ '1044267181-algorand' │ 'Algorand' │ 'folks-finance-lending' │ 'OPUL'  │  141257.71  │   180647.66    │    39389.95    │  0.66   │     3.43      │  0   │           │              │
    │    6    │ '971370097-algorand'  │ 'Algorand' │ 'folks-finance-lending' │ 'GALGO' │ 11253835.62 │  11253835.62   │       0        │    0    │       0       │ 0.65 │           │              │
    │    7    │ '971373361-algorand'  │ 'Algorand' │ 'folks-finance-lending' │ 'GOBTC' │  204019.62  │   204956.05    │     936.43     │    0    │     0.06      │ 0.7  │           │              │
    │    8    │ '971372237-algorand'  │ 'Algorand' │ 'folks-finance-lending' │ 'USDC'  │ 1180620.12  │   1680653.92   │    500033.8    │  1.01   │      3.8      │ 0.8  │   15.92   │   [ '0' ]    │
    │    9    │ '971372700-algorand'  │ 'Algorand' │ 'folks-finance-lending' │ 'USDT'  │  437868.03  │   490361.31    │    52493.27    │  0.24   │     2.49      │  0   │   19.87   │   [ '0' ]    │
    └─────────┴───────────────────────┴────────────┴─────────────────────────┴─────────┴─────────────┴────────────────┴────────────────┴─────────┴───────────────┴──────┴───────────┴──────────────┘
    This adapter contains some pools with <10k TVL, these pools won't be shown in DefiLlama

**Comment by slasher125** [comment]
ty sir

**Pull request merged**